src/main/java/com/epam/emotionalhelp/controller/dto/SubcategoryContainerDto.java

```diff
@@ -7,6 +7,7 @@
 
 @Data
 @RequiredArgsConstructor
+@Deprecated
```

Comment:
```suggestion
/**
 * This class can be used to store emotions subcategory details.
 *
 * @see SubcategoryDto
 * @deprecated to be removed in future releases
 */
@Data
@RequiredArgsConstructor
@Deprecated(since = "1.0", forRemoval = true)
```